src/content/self-hosted/install/releases.mdx

```diff
@@ -8,11 +8,80 @@ id: releases
 ## 1.7.0
 TBD
 
+### Removal Notice
+- Kubernetes [1.21](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.21.md) is no longer supported.
+
+### Deprecation Notice
+
+`cloud.provider` and `cloud.secret` helm chart values are deprecated and they will be removed in the future 2.x release. Now, [`registry.storage.provider`](self-hosted/administration/configuration.mdx#storage) and [`registry.secret`](self-hosted/administration/configuration.mdx#registry) should be used instead as they configure settings for the registry external storage. This change is backward compatible and it will work with the old keys (if they are provided) until they are removed.
+
+So, imagine you have this configuration:
+
+```yaml
+cloud:
+  secret:
+    name: "okteto-cloud-secret"
+    key: "key"
+    token: "token"
+  provider:
+    aws:
+      enabled: true
+      region: us-west-2
+      bucket: okteto-private-images
+      iam:
+        accessKeyID: "XXXXXX"
+```
+
+Now, it should be changed to and remove the config under `cloud`:
+
+```yaml
+registry:
+  secret:
+    name: "okteto-cloud-secret"
+    accessKey: "key"
+    secretKey: "token"
+  storage:
+    provider:
+      aws:
+        enabled: true
+        region: us-west-2
+        bucket: okteto-private-images
+        iam:
+          accessKeyID: "XXXXXX"
+```
+
 ### Features
 
 - Support for the annotation [dev.okteto.com/endpoints](cloud/ssl.mdx#customizing-the-endpoints-shown-in-the-okteto-ui) to customize the endpoints shown in the Okteto UI.
 - Support for configuring the [service account](self-hosted/administration/configuration.mdx#buildkit) of buildkit.
 - Support for [token-based authentication](self-hosted/administration/configuration.mdx#token) to simplify the installation of Okteto Self-Hosted.
+- [Personal Access Tokens](cloud/personal-access-tokens.mdx) now have expiration date. They will automatically expire after 180 days. For existing tokens, the expiration date is set to 180 days from when your instance is upgraded to 1.7 or newer versions
+- Upgraded the Okteto CLI version to [2.14.1](https://github.com/okteto/okteto/releases/tag/2.14.1)
+- Upgraded ingress-nginx to [4.3.0](https://github.com/kubernetes/ingress-nginx/releases/tag/helm-chart-4.3.0)
+- Allow to delete external resources individually from the UI
+- Default installation of Okteto doesn't require an external storage for the registry configured. It uses filesystem storage by default
+- Default installation of Okteto doesn't require a PVC for [BuildKit](self-hosted/administration/configuration.mdx#token). It uses [filesystem storage](self-hosted/administration/configuration.mdx#filesystem) by default
+
+### Improvements
+
+- When `buildkit.ingress.enabled` is true, BuildKit is exposed in the `buildkit.service.port` instead of port 1234. This makes easier to use other ingress controller.
```

Comment:
```suggestion
- When `buildkit.ingress.enabled` is true, BuildKit is exposed in the `buildkit.service.port` instead of port 1234. This makes it easier to use your with other ingress controllers
```